TAKAware/Screens/TAKAwareApp.swift

```diff
@@ -55,12 +55,15 @@ struct TAKTrackerApp: App {
                     .onChange(of: scenePhase) { newPhase in
                         if newPhase == .inactive {
                             TAKLogger.debug("[ScenePhase] Moving to inactive")
+                            NotificationCenter.default.post(name: Notification.Name(AppConstants.NOTIFY_APP_INACTIVE), object: nil)
```

Comment:
Was hoping we could use ScenePhase in MapView, but ScenePhase was always `Inactive` so chose to do a NotificationCenter post and respond to that instead